README.md

```diff
@@ -529,6 +529,22 @@ depot version
 
 PR contributions are welcome! The CLI codebase is evolving rapidly, but we are happy to work with you on your contribution.
 
+### Building from Source
+
+To build the CLI from source, you'll need to have Go installed. You can then build the binary by running:
+
+```shell
+make bin/depot
+```
+
+This will create the `depot` executable at `bin/depot`.
+
+Alternatively, you can run the `go build` command directly:
+
+```shell
+go build -o bin/depot ./cmd/depot
+```
```

Comment:
```suggestion
```
Honestly maybe get rid of this. Better not to hardcode this and then when this changes the docs are wrong